### PR TITLE
Fixed v01.06.01 DB Migration

### PR DESCRIPTION
**Pull request by Fosol**
The DB Migration v01.06.01 scripts were invalid.  I have updated them to resolve the issue.

You may not need to refresh your DB, or perhaps you may.

### CODE REVIEW

File: backend/dal/Migrations/v01.06.01/Down/PreDown/RemoveRolesAndClaims.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
-DELETE FROM dbo.[RoleClaims] where RoleId IN ('6238c23d-933e-4c4a-954e-48bae3ce2080', '5C6CEA5B-9B7C-47E8-852C-693E90ED815E', '090bc341-af0f-45c9-8e0a-6d99024b52c1');
-DELETE FROM dbo.[Roles] where Id = '5C6CEA5B-9B7C-47E8-852C-693E90ED815E';
-DELETE FROM dbo.[Claims] where Id = '81DED21C-ED32-4694-8F33-79EF17833F2B';
\ No newline at end of file
+DELETE FROM dbo.[RoleClaims] where RoleId = '5c6cea5b-9b7c-47e8-852c-693e90ed815e';
```
Comment (Fosol) [inline backend/dal/Migrations/v01.06.01/Down/PreDown/RemoveRolesAndClaims.sql:1]: Correctly delete the appropriate `RoleClaims`

File: backend/dal/Migrations/v01.06.01/Up/PostUp/AddRoleAndClaims.sql [hunk 2]
```diff
@@ -13,29 +13,31 @@ INSERT INTO
...
     (
-        '5C6CEA5B-9B7C-47E8-852C-693E90ED815E',
+        '5c6cea5b-9b7c-47e8-852c-693e90ed815e',
         'SRES Financial Reporter',
         'The SRES Financial Reporter can view, create, and delete non-final SPL reports.',
         0,
         0,
         0,
-        '5C6CEA5B-9B7C-47E8-852C-693E90ED815E'
+        '5c6cea5b-9b7c-47e8-852c-693e90ed815e'
```
Comment (Fosol) [inline backend/dal/Migrations/v01.06.01/Up/PostUp/AddRoleAndClaims.sql:22]: Case is important to Keycloak

File: backend/dal/Migrations/v01.06.01/Up/PostUp/AddRoleAndClaims.sql [hunk 3]
```diff
@@ -44,14 +46,14 @@ INSERT INTO dbo.[RoleClaims]
     )
 VALUES
 (
-    '6238c23d-933e-4c4a-954e-48bae3ce2080'     -- SRES Financial Manager
-    , '81DED21C-ED32-4694-8F33-79EF17833F2B'   -- reports-spl-admin
+    'd416f362-1e6f-4e24-a561-c6bb45a35194'     -- SRES Financial Manager
```
Comment (Fosol) [inline backend/dal/Migrations/v01.06.01/Up/PostUp/AddRoleAndClaims.sql:49]: Corrected invalid GUID values

**Review by LawrenceLau2020 — APPROVED** [review]
makes sense now

**Pull request merged**